scripts/cert_generation.sh

```diff
@@ -306,6 +336,7 @@ if [ ! -s "${WORKING_DIR}/kibana-session-secret" ] ; then
 fi
 
 if [ -f ${WORKING_DIR}/action.log ] ; then
-  cat ${WORKING_DIR}/action.log
+  sed -i ':a;N;$!ba;s/\n//g' ${WORKING_DIR}/action.log
```

Comment:
can you add a comment to what this sed line is doing?

Reply:
yes.  this is condensing the output and removing newlines.  Not sure if its needed in order to properly consume on the golang side...